reports/menubar-app-to-check-who-is-on-vacation.md

```diff
@@ -0,0 +1,108 @@
+# Menubar app to check who is on vacation
+
+The corresponding challenge is [#92](https://github.com/SolidLabResearch/Challenges/issues/92).
+
+## Problem
+<!--
+You can reuse the pitch of the challenge, but check if you need to make changes.
+For example, it might happen that the approved solution does more than what the original pitch requested.
+-->
+
+Manually checking your calendar to quickly see who is on vacation is a hassle. 
+It would be easier to have menubar app that shows in a glance who is on vacation today.
+
+## Approved solution
```

Comment:
I would expect a description of the technical choices, either here or in the readme of the app

Reply:
What technical choices do you mean here? Programming language? Frameworks?

Reply:
Added at https://github.com/SolidLabResearch/Challenges/pull/108/commits/8a208ff843c9d4ed66cea2f78afe3643b9f42481. Is this what you expect?

Reply:
Oh yes perfect!